### DISCUSSION TITLE
SSH remote command hangs.

### DISCUSSION THREAD

**Discussion by murrellr** [openssl/openssl#23712] [General Discussion]
I'm trying to update our system from Linux 4.14 and Debian stretch to Linux 6.2.0 and Debian bullseye.  I have locally built the Linux kernel using a custom configuration and a stripped down Debian bullseye image.  This runs an a small ARM embedded system.

I'm having problems running remote commands through SSH.  With our old image I could run this command from Windows 10 CMD:
```
C:\Devel>ssh root@172.23.165.1 ls
root@172.23.165.1's password:
BSP-VERSION.txt README.txt debug.sh initialized
Connection to 172.23.165.1 closed.
```
Now, with the new image, I get this:
```
C:\Devel>ssh root@172.23.165.1 ls
root@172.23.165.1's password:
stty: 'standard input': Inappropriate ioctl for device
stty: 'standard input': Inappropriate ioctl for device
[18t
```
The program hangs and I have to type Ctrl/C to exit.  I can add the "-t" argument and it works.  The problem I that I am using a third party Windows remote debugging product that internally opens an SSH connection to the target system.  It hangs when establishing a connection to the target system.  What changed in sshd that prevents this from working?

New Linux sshd version: OpenSSH_8.4p1 Debian-5+deb11u3, OpenSSL 1.1.1w  11 Sep 2023
Old sshd version: OpenSSH_7.4p1 Debian-10+deb9u7, OpenSSL 1.0.2u  20 Dec 2019

**Comment by kroeckx** [comment]
This has nothing to do with OpenSSL. This is most likely something that is executed as part of setting up the shell environment, so something like bashrc is doing. Did you upgrade everything?

**Comment by tom-cosgrove-arm** [comment]
To be more specific, this appears to be a question about OpenSSH, but has been asked on a discussion forum about OpenSSL - different projects

**Comment by murrellr** [comment]
I surmised this after clearing my mind and going to bed.  I suddenly realized that the "[18t" string at the end of problem session is part of a terminal control escape sequence and the session might be locked because it is waiting for user input.  And I did upgrade everything on the move to Debian bullseye.  Thank you for your input.  And my apologies for posting to the wrong forum.
